src/store/reducers/appReducer.js

```diff
@@ -129,9 +160,14 @@ const initialState = {
   logo: svgs.contoso,
   name: 'companyName',
   isDefaultLogo: true,
-  azureMapsKey: '',
   deviceGroupFlyoutIsOpen: false,
   timeInterval: 'PT1H',
+  settings: {
+    azureMapsKey: '',
+    description: '',
+    name: '',
+    diagnosticsOptIn: true
```

Comment:
The default for a boolean should be false.  I would also question being *in* for this as the default. That would mean we're assuming they are OK with the privacy statement when they haven't actually seen it yet (or been given that opportunity), right? That defeats the purpose of having to opt in.

Reply:
This is how the feaature has been implemented in DS. I asked @ppathan about it as well.

Reply:
Hrmm, still feels backwards to me.